mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -38,6 +38,11 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	METHOD method_30267 isSectionHidden (ILnet/minecraft/class_1799$class_5422;)Z
 		ARG 0 flags
 	METHOD method_30268 addHideFlag (Lnet/minecraft/class_1799$class_5422;)V
+	METHOD method_31575 onStackClicked (Lnet/minecraft/class_1799;Lnet/minecraft/class_5536;Lnet/minecraft/class_1661;)Z
+	METHOD method_31576 onClicked (Lnet/minecraft/class_1799;Lnet/minecraft/class_5536;Lnet/minecraft/class_1661;)Z
+	METHOD method_31578 showItemBar ()Z
```

Comment:
These mappings should probably match the updated ones in `Item`

Reply:
Matched to the mappings in `Item`